core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -155,4 +166,6 @@ class ExpressionGenerator(val callMapper: ir.CallMapper = new ir.CallMapper())
   private def orNull(option: Option[String]): String = option.getOrElse("NULL")
 
   private def doubleQuote(s: String): String = s""""$s""""
+
+  private def singleQuote(s: String): String = s"'$s'"
```

Comment:
let's revert this for now.

Reply:
why?

Reply:
From the [documentation](https://docs.databricks.com/en/sql/language-manual/data-types/string-type.html)

> While you can use double quotes (") instead of single quotes (') to delimit a string literal, this is discouraged as it is not standard SQL.